features/step_definitions/networking.rb

```diff
@@ -1303,3 +1303,12 @@
   default_network = network_operator.default_network(user: admin)
   raise "env doesn't have IPSec enabled" unless default_network["ipsecConfig"]
 end
+
+Given /^the node's active nmcli connection is stored in the#{OPT_SYM} clipboard$/ do |cb_name|
+  ensure_admin_tagged
+  cb_name = "nmcli_active_con_uuid" unless cb_name
```

Comment:
@kedark3 perhaps shorten the default cb_name

Reply:
ok I will make it shorter :)